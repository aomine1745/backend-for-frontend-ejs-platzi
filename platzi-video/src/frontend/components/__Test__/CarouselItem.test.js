import React from 'react';
import { create } from 'react-test-renderer';
import ProviderMock from '../../__mocks__/providerMock';
import CarouselItem from '../CarouselItem';

describe('Carousel Item component', () => {
  test('Match snapshot', () => {
    const carouselItem = create(
      <ProviderMock>
        <CarouselItem />
      </ProviderMock>,
    );
    expect(carouselItem.toJSON()).toMatchSnapshot();
  });
});
