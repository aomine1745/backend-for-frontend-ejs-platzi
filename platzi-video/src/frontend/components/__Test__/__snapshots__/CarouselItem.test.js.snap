// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Item component Match snapshot 1`] = `
<div
  className="Carousel__item"
>
  <img
    className="Carousel__item-img"
  />
  <div
    className="Carousel__details"
  >
    <div
      className="Carousel__details-actionBox"
    >
      <a
        href="/player/undefined"
        onClick={[Function]}
      >
        <img
          alt="Play Icon"
          className="Carousel__details-action"
          src="test-file-stubs"
        />
      </a>
      <img
        alt="Plus Icon"
        className="Carousel__details-action"
        onClick={[Function]}
        src="test-file-stubs"
      />
    </div>
    <p
      className="Carousel__details-title"
    />
    <p
      className="Carousel__details-subtitle"
    >
      undefined undefined undefined
    </p>
  </div>
</div>
`;
